src/main/webapp/WEB-INF/view/logout.jsp

```diff
@@ -30,5 +31,8 @@
     <form action ="/logout" method = "POST">
       <button type="submit">Yes</button>
     </form>
-    <p>no</p>
+
+    <a href ="/">
```

Comment:
Have either 
<a ..> <button> No </botton> </a> 

or 

<a..>
    <button>No</button>
</a> 

for readability purposes